app/views/shared/_date_time_form_part.html.erb

```diff
@@ -1,11 +1,11 @@
 <% today, tomorrow, initial_date, initial_date_type = data_chooser_setup(@defence_request.send(attribute), set_default_date) %>
 <div class="form-group">
   <fieldset class="inline time-select date-chooser">
-    <% if show_label %>
+    <%- if defined?(show_label) && show_label %>
```

Comment:
`defined?` smells bad, can we use `local_assigns.fetch :show_label, false` at the top of the file instead? Or just explicitly pass in the variable to each render call.